imap_processing/ultra/l1b/lookup_utils.py

```diff
@@ -0,0 +1,160 @@
+"""Contains tools for lookup tables for l1b."""
+
+import numpy as np
+import pandas as pd
+
+from imap_processing import imap_module_directory
+
+base_path = imap_module_directory / "ultra" / "lookup_tables"
```

Comment:
Global constants should be all uppercase. Optionally starting with "_" like you did on the others below indicates it is a "private" global only to be used this module.
```suggestion
BASE_PATH = imap_module_directory / "ultra" / "lookup_tables"
```